notices.json

```diff
@@ -38,21 +38,21 @@
     }
   },
   {
-    "id": "server_upgrade_v6.5",
+    "id": "server_upgrade_v6.6",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<6.5"],
+      "serverVersion": ["<6.6"],
       "instanceType": "onprem",
-      "displayDate": ">= 2022-03-21T00:00:00Z"
+      "displayDate": ">= 2022-04-18T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
-        "title": "Mattermost 6.5 is here!",
-        "description": "Mattermost v6.5 includes custom groups, cross-team channel navigation, playbook import, export, and duplication, improved board sharing, Bitbucket integration, workplace optimization, improved onboarding experiences and Persian language support. [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
+        "title": "Mattermost 6.6 is here!",
+        "description": "Mattermost v6.6 includes triggers and actions on channel messages, retrospective metrics on playbooks and the general availability of Apps Framework. [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
```

Comment:
0/5 if we want to include links to more resources for each of these items where applicable? ie the GA of the apps framework.

@amyblais perhaps it's worth reaching out to the PMs of each of these highlights to see if there's extra content to link to that would make these notices more impactful?

Reply:
And perhaps those PMs should also be on review for the notice? ^